index.html

```diff
@@ -2,268 +2,411 @@
+              <div class="project-details">
+                <p>
+                  A daily selection of privately <br />personalized reads; no
+                  accounts or sign-ups <br />required. has been the industry's
+                  standard <br />
+                  dummy text ever since the 1500s, when an <br />
+                  unknown printer took a standard dummy text.
+                </p>
+              </div>
               <div class="project-links">
                 <ul>
-                  <li> css </li>
-                  <li> html</li>
+                  <li>css</li>
+                  <li>html</li>
                   <li>bootstrap</li>
-                  <li>Ruby </li>
+                  <li>Ruby</li>
                 </ul>
               </div>
-              <button class="bttn" type="button">See Project</button>
+              <button class="bttn raise-modal" id="003" type="button">See project</button>
             </div>
           </div>
+        </div>
+        <div class="projects-seection left-img">
+          <a class="" href="#"
+            ><img src="images/img-placeholder.png" alt="Project 4"
+          /></a>
+          <div class="project-text">
+            <h2 class="post-tittle">Multi-Post Stories</h2>
+            <div class="support-text">
+              <div class="project-details">
+                <p>
+                  A daily selection of privately <br />personalized reads; no
+                  accounts or sign-ups <br />required. has been the industry's
+                  standard <br />
+                  dummy text ever since the 1500s, when an <br />
+                  unknown printer took a standard dummy text.
+                </p>
+              </div>
+              <div class="project-links">
+                <ul>
+                  <li>css</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
+              <button class="bttn raise-modal" type="button">See Project</button>
+            </div>
           </div>
-      </div>  
+        </div>
+      </div>
     </section>
 
     <section class="contact-container" id="contact">
       <div class="frame42">
-        <h2 class="contact-tittle2"> Let’s chat!</h2>
+        <h2 class="contact-tittle2">Let’s chat!</h2>
         <h2 class="contact-tittle1">CONTACT US</h2>
         <p>
-          If you have an application you are
-          interested in developing, a feature that you <br>
-          need built or a project that needs coding.
-          I’d love to help with it
+          If you have an application you are interested in developing, a feature
+          that you <br />
+          need built or a project that needs coding. I’d love to help with it
         </p>
       </div>
-      <form method="post" action="https://formspree.io/f/mjvlplnjl">
+      <form method="post" action="https://formspree.io/f/mjvlplnjl" id="myform">
```

Comment:
- [ ] [OPTIONAL] The form can't be submitted, kindly check your id for form spree so the form is working properly